phpBB/styles/prosilver/theme/icons.css

```diff
@@ -9,7 +9,7 @@
  * Just change the name of the font after the 14/1 to the name of
  * the font you wish to use.
  */
-.icon, .button .icon  {
+.icon, .button .icon, blockquote cite:before, .uncited:before  {
```

Comment:
Thanks. I thought I had checked for spacing issues. This relatively new contributor continues to learn  and thanks the phpBB team for their assistance, guidance and patience.